Cargo.lock

```diff
@@ -8811,7 +8810,7 @@ version = "1.6.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "97fee6b57c6a41524a810daee9286c02d7752c4253064d0b05472833a438f675"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
How come it's getting rolled back to a previous version, when it's using 1.0.0 everywhere else?

Reply:
It's by design. Try `cargo update openssl`.

Reply:
Done

Reply:
I give up. Feel free to take over from here :)